src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -695,7 +695,7 @@ static string GetStyledText(TaggedText taggedText, bool isInCodeBlock)
                     TaggedTextStyle.Strong => $"**{text}**",
                     TaggedTextStyle.Emphasis => $"_{text}_",
                     TaggedTextStyle.Underline => $"<u>{text}</u>",
-                    TaggedTextStyle.Code => $"`{text}`",
+                    TaggedTextStyle.Code or TaggedTextStyle.InlineCode => $"`{text}`",
```

Comment:
Few lines above this, there is:

```csharp
                if (!isInCodeBlock)
                    text = text.Replace(" ", "&nbsp;");
```

This doesn't feel correct. See the following test:

https://github.com/dibarbet/roslyn/blob/0ef88b0a686892546b57e721ba65738872bf928d/src/Features/LanguageServer/ProtocolUnitTests/Hover/HoverTests.cs#L195

@sharwell Do you agree here? Do you want me to fix LSP in this PR or make that a separate PR?

cc: @dibarbet

Reply:
Separate is fine.